tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -423,10 +438,18 @@ Future _runCommand(ArgResults results, ArgParser parser) async {
 }
 
 void _printUsage(ArgParser parser) {
-  print('${Colorize('Usage:')..bold()} serverpod <command> [arguments]\n');
-  print('');
-  print('${Colorize('COMMANDS')..bold()}');
-  print('');
+  log.info(
```

Comment:
It would be nicer to encapsulate the colorization in the logger and making it possible to pass arguments to the log method.

Reply:
Removing them for now and then we can improve it later. Since we are already looking at moving to CommandRunner it might be good to use the build in usage string.